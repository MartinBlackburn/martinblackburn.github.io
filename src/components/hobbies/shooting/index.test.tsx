// libraries
import { render } from "@testing-library/react";

// component under test
import ShootingText from "./";

describe("ShootingText component", () => {
    test("renders correctly", () => {
        const { asFragment } = render(<ShootingText />);

        expect(asFragment()).toMatchSnapshot();
    });
});
