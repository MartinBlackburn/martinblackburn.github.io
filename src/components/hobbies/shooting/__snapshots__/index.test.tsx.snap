// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShootingText component renders correctly 1`] = `
<DocumentFragment>
  <p>
    I recently joined 
    <a
      href="http://www.yorkrifle.club/"
    >
      York rifle club
    </a>
    . It's still early days and I'm not a great shot, but it's a fun sport and everyone is incredible friendly and helpful. Looking forward to seeing how much I improve over the next few months.
  </p>
</DocumentFragment>
`;
